graylog2-server/src/main/java/org/graylog2/alarmcallbacks/AlarmCallbackHistoryImpl.java

```diff
@@ -31,6 +32,7 @@
 import org.mongojack.ObjectId;
 
 @AutoValue
+@WithBeanGetter
```

Comment:
Yes, but I didn't want to check each class individually and used some `sed` magic to replace `@AutoValue` with `@AutoValue\n@WithBeanGetters` (minus those classes which already contained getters). 😉